services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -1473,8 +1475,18 @@ public String putRelativeFile(EditorConfig config,
    */
   protected String createFile(Node parent, String filename, InputStream data) throws RepositoryException,
                                                                               FileExtensionNotFoundException {
-    // Add node
-    Node addedNode = parent.addNode(filename, Utils.NT_FILE);
+    String nodeName = filename;
+    boolean fileCreated = false;
+    Node addedNode = null;
+    int count = 0;
+    do {
+      try {
+        addedNode = parent.addNode(nodeName, FCKUtils.NT_FILE);
```

Comment:
Don't use FCK stuff, it's separate UI feature only, we already `NodetypeConstant.NT_FILE`.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/36/commits/c3008ded304dd80e69fdb58e9227400abbea1ba5